packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -117,8 +118,9 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
             MeasureToolDefinitions.clearMeasurementsToolCommand,
             {
               isHidden: new ConditionalBooleanValue(
-                () => isSheetViewActive() || !MeasurementUIEvents.isClearMeasurementButtonVisible,
+                () => !MeasurementUIEvents.isClearMeasurementButtonVisible,
```

Comment:
Is this something that can wait until appui 5.x is released?
With the refactor that deprecates `provideToolbarItems` in favor of `getToolbarItems`, the consuming application will have the freedom to add or override the `isHidden` at will, making this change useless.
If it's needed right now, I suppose we can squeeze it in and remove it later on.

Reply:
Well, let's comment on the original PR, I was trying to validate if this was an issue with organisation. I'm closing this PR now.